redskyctl/internal/commands/authorize_cluster/generator.go

```diff
@@ -182,3 +192,30 @@ func (o *GeneratorOptions) clientInfo(ctx context.Context, ctrl *config.Controll
 	}
 	return o.Config.RegisterClient(ctx, client)
 }
+
+type helmValuesPrinter struct {
+}
+
+func (h helmValuesPrinter) PrintObj(i interface{}, w io.Writer) error {
+	if secret, ok := i.(*corev1.Secret); ok {
```

Comment:
nit, Could we return early here?

```
secret, ok := i.(*corev1.Secret)
if !ok {
  return nil
}

vals ....
```

Reply:
Yeah, I should probably be better about that in general...